runhouse/utils.py

```diff
@@ -717,3 +726,87 @@ class StatusColors(str, Enum):
     @classmethod
     def get_status_color(cls, status: str):
         return getattr(cls, status.upper()).value
+
+
+def create_output_table(
+    total_clusters: int,
+    running_clusters: int,
+    displayed_clusters: int,
+    filters_requested: bool,
+):
+    """The cluster list is printed as a table, this method creates it."""
+    from runhouse.globals import rns_client
+
+    displayed_running_clusters = (
+        running_clusters
+        if running_clusters < displayed_clusters
+        else displayed_clusters
+    )
+    table_title = f"[bold cyan]Clusters for {rns_client.username} (Running: {displayed_running_clusters}/{running_clusters}, Total Displayed: {displayed_clusters}/{total_clusters})[/bold cyan]"
+
+    table = Table(title=table_title)
+
+    if not filters_requested:
+        table.caption = f"[reset]Showing clusters that were active in the last {int(LAST_ACTIVE_AT_TIMEFRAME / HOUR)} hours."
+        table.caption_justify = "left"
+
+    if displayed_clusters == MAX_CLUSTERS_DISPLAY:
+        link_to_clusters_in_den = f"[reset]Full list of clusters can be found in https://www.run.house/resources?folder={rns_client.username}&type=cluster. Please make sure yor as signed as {rns_client.username} before following the link."
```

Comment:
if the user isn't logged it they will be prompted to log in via the UI anyways
```suggestion
        link_to_clusters_in_den = f"[reset]The full cluster list can be viewed at https://www.run.house/resources?folder={rns_client.username}&type=cluster."
```